applications/RANSApplication/custom_elements/incompressible_potential_flow_velocity_element.cpp

```diff
@@ -58,7 +58,7 @@ void IncompressiblePotentialFlowVelocityElement<TDim, TNumNodes>::GetValueOnInte
             rValues[g] = velocity;
         }
     } else {
-        KRATOS_ERROR << "GetValueOnIntegrationPoints for variable "
+        KRATOS_ERROR << "CalculateOnIntegrationPoints for variable "
                      << rVariable.Name() << " not defined for " << this->Info();
```

Comment:
I think that you should call the base class in here. In any case this is out of the scope of this PR.

Reply:
I will do it... thanks for pointing out :)